### PR TITLE
Update index.zh-CN.md

### PR DESCRIPTION
**Pull request by quicksand-1**
优化 getCheckboxProps 文档

First of all, thank you for your contribution! 😄

New feature please send pull request to feature branch, and rest to master branch.
Pull request will be merged after one of collaborators approve.
Please makes sure that these form are filled before submitting your pull request, thank you!

[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### This is a ...

- [ ] New feature
- [ ] Bug fix
- [x] Site / document update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Branch merge
- [ ] Other (about what?)

### What's the background?

> 1. Describe the source of requirement.
> 2. Resolve what problem.
> 3. Related issue link.

### API Realization (Optional if not new feature)

> 1. Basic thought of solution and other optional proposal.
> 2. List final API realization and usage sample.
> 3. GIF or snapshot should be provided if includes UI/interactive modification.

### What's the effect? (Optional if not new feature)

> 1. Does this PR affect user? Which part will be affected?
> 2. What will say in changelog?
> 3. Does this PR contains potential break change or other risk?

### Changelog description (Optional if not new feature)

> 1. English description
> 2. Chinese description (optional)

### Self Check before Merge

- [x] Doc is updated/provided or not needed
- [ ] Demo is updated/provided or not needed
- [ ] TypeScript definition is updated/provided or not needed
- [ ] Changelog is provided or not needed

### Additional Plan? (Optional if not new feature)

> If this PR related with other PR or following info. You can type here.

### CODE REVIEW

File: components/table/index.zh-CN.md [hunk 1]
```diff
@@ -164,7 +164,7 @@ const columns = [{
 | columnWidth | 自定义列表选择框宽度 | string\|number | - |
 | columnTitle | 自定义列表选择框标题 | string\|React.ReactNode | - |
 | fixed | 把选择框列固定在左边 | boolean | - |
-| getCheckboxProps | 选择框的默认属性配置 | Function(record) | - |
+| getCheckboxProps | 选择框的默认属性配置，参照 Checkbox 属性。使用该属性，最后需返回 { disabled } | Function(record) | - |
```
Comment (afc163) [inline components/table/index.zh-CN.md]: 感觉这是个 bug。

Comment (long-zhuge) [reply components/table/index.zh-CN.md]: @afc163 
image
image

Comment (zombieJ) [reply components/table/index.zh-CN.md]: 看起来是个 bug，有兴趣发个 PR 修一下吗？ @long-zhuge

**Comment by zombieJ** [comment]
ref: #15224

**Pull request closed without merging**